src/styles.css

```diff
@@ -0,0 +1,113 @@
+body {
+  margin: 0;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 0;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.card__product-code {
+  display: flex;
+  margin-bottom: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  text-align: center;
+  color: #616070;
+  line-height: 14px;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url("./images/star.svg");
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.colored {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
There should be 2 different style sheets for `.card ` and for `.stars` blocks
[checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) p.11

Also, the `.stars` block must be operational in case you change block modifier.
[checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) p.4